python/ray/autoscaler/aws/node_provider.py

```diff
@@ -222,7 +226,9 @@ def create_node(self, node_config, tags, count):
 
                 self.ec2.meta.client.start_instances(
                     InstanceIds=reuse_node_ids)
+                reuse_tags = dict(tags)
                 for node_id in reuse_node_ids:
+                    reuse_tags[TAG_RAY_FIRST_BOOT] = False
```

Comment:
The reuse tags dict doesn't seem to be ever used.

Reply:
thanks, this is a bug